app/views/notifications/_emergency_banner.html.erb

```diff
@@ -1,9 +1,9 @@
-<section id="emergency-banner-notification" class="<%= banner.campaign_class %>">
+<section id="emergency-banner-notification" class="<%= banner.campaign_class %>" role="banner">
```

Comment:
FYI, `<section>` has the implicit aria role of `region`, so it should either be one or the other... or both if we want to cater for older screen readers, as we do with other elements, e.g. `header`.